README.md

```diff
@@ -8,9 +8,11 @@ Dependencies
 
 * [Composer](https://getcomposer.org/)
 * [npm](https://www.npmjs.com/)
-* PHP 7
+* [PHP 7](https://www.php.net/)
+* [Docker](https://www.docker.com/)
+* [Docker-compose](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-comp
 
-Installation
+Manual Installation (may or may not work - better use docker)
```

Comment:
I would move these around, so that the docker instructions are above the manual ones